remoting/subsystem/src/main/java/org/jboss/as/remoting/LocalOutboundConnectionWriteHandler.java

```diff
@@ -57,12 +56,11 @@ protected void revertUpdateToRuntime(OperationContext context, ModelNode operati
         if (handback != null && !handback.booleanValue()) {
             final ModelNode restored = Resource.Tools.readModel(context.readResource(PathAddress.EMPTY_ADDRESS));
             restored.get(attributeName).set(valueToRestore);
-            applyModelToRuntime(context, operation, restored);
+            applyModelToRuntime(context, operation);
```

Comment:
this change is confusing.
At first glance, I thought that the change  would not take into account the `restored` model but it turns out that the parameters end up not being used in the `LocalOutboundConnectionService` constructor.
Is that correct?

Reply:
Correct, I eliminated many unused fields in second commit of this PR.